README.md

```diff
@@ -10,7 +10,13 @@ Build and run the website locally:
 1. Run the following npm commands:
 
 ```bash
+git clone https://github.com/fermyon/developer.git
```

Comment:
Consider dropping these first two lines - people will often be cloning from their own forks, and they kind of bury the key bit of information about doing the hub build.  Or break this out into sections a la

```
# Clone the site locally
git clone ...
cd ...

# Build the site
... cd, npm install, cd, install, build ...

# Run the site and reload when content changes
spin watch
```

Regardless, consider removing "npm" on line 10.